dsl/git_push_test.jmx

```diff
@@ -0,0 +1,254 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="2.8" jmeter="2.13 r1665067">
```

Comment:
Sorry, didn't see that. I'm no clearer about how to read it though. Is it these sections that have been added?
```
+          <elementProp name="SystemSampler.arguments" elementType="Arguments" guiclass="ArgumentsPanel" testclass="Arguments" testname="OsProcessSampler" enabled="true">
+            <collectionProp name="Arguments.arguments">
+              <elementProp name="" elementType="Argument">
+                <stringProp name="Argument.name"></stringProp>
+                <stringProp name="Argument.value">push</stringProp>
+                <stringProp name="Argument.metadata">=</stringProp>
+              </elementProp>
+              <elementProp name="" elementType="Argument">
+                <stringProp name="Argument.name"></stringProp>
+                <stringProp name="Argument.value">git@ci-gandalf.tsuru2.paas.alphagov.co.uk:davas-${app_id}.git</stringProp>
+                <stringProp name="Argument.metadata">=</stringProp>
+              </elementProp>
+              <elementProp name="" elementType="Argument">
+                <stringProp name="Argument.name"></stringProp>
+                <stringProp name="Argument.value">master</stringProp>
+                <stringProp name="Argument.metadata">=</stringProp>
+              </elementProp>
+            </collectionProp>
+          </elementProp>
```

```
+          <elementProp name="SystemSampler.environment" elementType="Arguments" guiclass="ArgumentsPanel" testclass="Arguments" testname="OsProcessSampler" enabled="true">
+            <collectionProp name="Arguments.arguments">
+              <elementProp name="HOME" elementType="Argument">
+                <stringProp name="Argument.name">HOME</stringProp>
+                <stringProp name="Argument.value">/tmp/tsuru_tmp</stringProp>
+                <stringProp name="Argument.metadata">=</stringProp>
+              </elementProp>
+              <elementProp name="GIT_SSH" elementType="Argument">
+                <stringProp name="Argument.name">GIT_SSH</stringProp>
+                <stringProp name="Argument.value">/tmp/tsuru_tmp/ssh-wrapper</stringProp>
+                <stringProp name="Argument.metadata">=</stringProp>
+              </elementProp>
+            </collectionProp>
+          </elementProp>
```

Maybe list that in the README? I wouldn't have understood how to get from those bullet points to that XML.